code/controllers/subsystem/communications.dm

```diff
@@ -99,6 +99,14 @@ var/const/CLF_ENGI_FREQ = 1273
 var/const/CLF_MED_FREQ = 1274
 var/const/CLF_CCT_FREQ = 1275
 
+//Listening Bugs (1290-1294)
+var/const/BUG_A_FREQ = 1290
```

Comment:
why hardcode in 5 freqs?

Reply:
Removed three of them due to using existing frequencies on most bugs. 
Undefined frequencies last I checked have a habit of spitting out messages weirdly.